cbopensource/connectors/taxii/cybox_parse.py

```diff
@@ -62,6 +62,26 @@ def validate_md5sum(md5):
     return True
 
 
+def validate_sha256(sha256):
+    """
+    Validate md5sum
+    :param md5: md5sum to valiate
+    :return: True or False
+    """
+    if 64 != len(sha256):
+        logger.warn("Invalid sha256 length for sha256 {}".format(sha256))
+        return False
+    if not sha256.isalnum():
+        logger.warn("Malformed sha256 {} in IOC list".format(sha256))
+        return False
+    for c in "ghijklmnopqrstuvwxyz":
```

Comment:
What is "c"?  I'm already lost.

Reply:
"c" for "character" -- I read the expression as checking for the existence of characters in the string that would be illegal for SHA256 (I think there may be similar code in cbent, but I could be wrong)

Reply:
Yeah I think this is a questionable way to validate a sha256 hash i'm just repurposing the code that was used to validate md5 hashes and walking out the hash-length to be appropriate for sha256.